src/v2/models/AppState.js

```diff
@@ -77,6 +78,19 @@ export default Model.extend({
     return this.get('idx').neededToProceed.find((remediation) => remediation.name === formName);
   },
 
+  hasMoreThanOneAuthenticatorOption(formName) {
+    const form = this.hasRemediationObject(formName);
+    if (form) {
```

Comment:
nit: To avoid a large conditional block, consider returning when the following isn't met:

```js
if (!form || !form.value.find(value => value.name === 'authenticator')) {
  // return
}
```